lang/luasocket/Makefile

```diff
@@ -42,7 +42,7 @@ endef
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR)/ \
 		LIBDIR="$(TARGET_LDFLAGS)" \
-		CC="$(TARGET_CC) $(TARGET_CFLAGS) $(TARGET_CPPFLAGS) -std=gnu99" \
+		CC="$(TARGET_CC) $(TARGET_CFLAGS) $(TARGET_CPPFLAGS) $(FPIC)" \
```

Comment:
you should leave the std here, that's project code dependent, and removing it relies on the default std of our toolchains, which has changed, and might change again.

Reply:
From what I've seen, the only real impact is the change in inlining from 89 to 99. I've yet to see a difference elsewhere (not that many projects use c11 atomics).

In any case, I added -std=gnu89 and saw that it compiled fine.